### ISSUE TITLE
Cert-manager: Apply CRDs as part of Helm Installation

### ISSUE THREAD

**Issue by archups** [kubermatic/kubermatic#11255]
### Description of the feature you would like to add / User story

As a end user of KKP
I would be nice to handle cert-manager installation via Helm can be handled without explicit need for application of CRDs 
in order to simplify and use the cert-manager helm chart supported flag `installCRDs`.  

### Solution details

- It would be nice to handle at KKP Cert-manager chart's values.yaml level to add  `installCRDs`: `true` (Reference https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L53). So cert-manager installation via Helm can be handled without explicit need for application of CRDs on the Seed Cluster.

### Use cases

This is very common use-case for separate seed installation, where cert-manager installation is required for certificates management.

**Comment by embik** [comment]
> So cert-manager installation via Helm can be handled without explicit need for application of CRDs on the Seed Cluster.

Can you specify what you mean by this, is cert-manager even installed on Seeds? Or do you mean you're using the KKP chart for cert-manager to manually install it onto KKP Seeds? What's the use-case for that?

**Comment by archups** [comment]
yes when we have separate Master and Seed clusters , we are using KKP cert-manager helm chart on the seed cluster for installation.

**Comment by embik** [comment]
As far as I understand you should be able to set the `installCRDs` flag for the upstream chart already if you update the `values.yaml` file you pass to `helm` for installing the KKP cert-manager chart. Does that not work?

**Comment by xrstf** [comment]
We specifrically had to move away from Helm managing the CRDs because _Helm never updates CRDs_, which was completely breaking the v1alpha1 => v1alpha2 => v1 migration of cert-manager. In order to never have this trouble ever again, we decided to setup the CRDs ourselves. Also because the installer is doing the migrations.

**Comment by archups** [comment]
> We specifrically had to move away from Helm managing the CRDs because _Helm never updates CRDs_, which was completely breaking the v1alpha1 => v1alpha2 => v1 migration of cert-manager. In order to never have this trouble ever again, we decided to setup the CRDs ourselves. Also because the installer is doing the migrations.

Got it now.. noted!
